extension.json

```diff
@@ -15,7 +15,8 @@
     "release-schema.json"
   ],
   "dependencies": [
-    "https://raw.githubusercontent.com/open-contracting-extensions/ocds_location_extension/master/extension.json"
+    "https://raw.githubusercontent.com/open-contracting-extensions/ocds_location_extension/master/extension.json",
+    "https://raw.githubusercontent.com/open-contracting-extensions/ocds_lots_extension/master/extension.json"
```

Comment:
I think it is not dependent, but you will need to set it under `testDependencies`.

Reply:
Could you explain the difference? Since this extension now adds a field to `Lot`, I thought it would be dependent on the lots extension. I can see that it is possible to patch the OCDS schema with this extension's schema without also applying the lots extension, but the result is kind of meaningless because the `Lot` definition is not referenced anywhere.

Reply:
From https://ocds-standard-development-handbook.readthedocs.io/en/latest/extensions/index.html

> Follow the instructions in the [extension template](https://github.com/open-contracting/standard_extension_template/blob/master/README.md).

> * Use dependencies if the extension $ref’erences another extension’s definitions.
> * Use testDependencies if the extension adds fields to another extension’s definitions.

When `testDependencies` is set, the standard-maintenance-scripts tests add those extensions (this avoids errors about the `Lot` definition not having a `title` or `description`).

When `dependencies` is set, all tools ought to add those extensions.

Since bidOpening can be used without lots, it does not depend on the Lot extension. (And as you say, it just ends up uselessly defining a field that will not be used, in that scenario).

Reply:
Ah, I see. Thanks. I looked in the extension template, the extension explorer documentation and the `extension.json` schema in the standard maintenance scripts repo... but not in the handbook!

Reply:
Done in d9bc16ab83f173d1e89d70b31c994a32d8be33cd

Reply:
Ah, we can maybe clean that up - not sure how to rearrange the content so it's more discoverable. The instructions in the handbook are (supposed to be) limited to OCP-authored extensions. testDependencies isn't mentioned anywhere else (I think) because no one else runs our tests.

Reply:
All good. I think it's reasonable to expect I would check the handbook! I tried to add the following sentence to https://github.com/open-contracting/standard-maintenance-scripts/tree/main/schema#extension-schemajson, but I don't have permission to create branches in that repo.

> Fields that are only relevant to OCP-authored extensions are documented in the [OCDS Development Handbook extensions documentation](https://ocds-standard-development-handbook.readthedocs.io/en/latest/extensions/index.html).

Reply:
I've added it now. You can create branches via a fork, just not on that repo itself.